polus-imagej-plugins/polus-imagej-image-integral-plugin/src/ij_converter.py

```diff
@@ -0,0 +1,118 @@
+
+'''
+A conversion utility built to convert abstract to primitive
+This works for Threshold Apply and Gaussian Filter functions
+Note change from jnius to jpype for handling conversions in ij_converter.py
+
+'''
+
+import imagej
+import logging
+import imglyb
+import jpype
+import numpy as np
+import scyjava
+
+# Initialize the logger
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("ij_converter")
+logger.setLevel(logging.INFO)
+
+ij = None
```

Comment:
This `ij` "global" makes me very uncomfortable. It's not readable since there is nothing to indicate it's being set in another file. I recommend you explicitly pass in `imagej_instance` or something as an argument to all functions that need it. Or define a class and this can be a property that is passed into the constructor.